Sources/Testing/EntryPoints/EntryPoint.swift

```diff
@@ -348,8 +354,14 @@ func parseCommandLineArguments(from args: [String]) throws -> __CommandLineArgum
       .filter { args[$0] == label && $0 < args.endIndex }
       .map { args[args.index(after: $0)] }
   }
-  result.filter = filterValues(forArgumentsWithLabel: "--filter")
-  result.skip = filterValues(forArgumentsWithLabel: "--skip")
+  let filter = filterValues(forArgumentsWithLabel: "--filter")
```

Comment:
Without this part of the change, any filters/skips set by the input JSON blob will be overwritten.